Backpack-SwiftUI/Calendar/Classes/BPKCalendar.swift

```diff
@@ -50,15 +49,13 @@ public struct BPKCalendar<DayAccessoryView: View>: View {
         calendar: Calendar,
         validRange: ClosedRange<Date>,
         initialMonthScroll: MonthScroll? = nil,
-        calendarAccessibilityConfiguration: CalendarAccessibilityConfiguration,
```

Comment:
This is now handled as it was before, as part of the binding itself, to not allow the API to lead to invalid states